integration_tests/pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsPage.ts

```diff
@@ -0,0 +1,43 @@
+import Page from '../page'
+
+export default class PrisonerChangeIncentiveLevelDetailsPage extends Page {
+  constructor() {
+    super(`Record John Smith’s incentive level`)
+  }
+
+  get pageHeading() {
+    return cy.get('h1')
+  }
+
+  get currentIncentiveLevel() {
+    return cy.get('[data-test="current-incentive-level"]')
+  }
+
+  get newLevel() {
+    return cy.get('[data-test="new-level"]')
+  }
+
+  get submitChange() {
+    return cy.get('[data-test="submit-change"]')
+  }
+
+  get formErrors() {
+    return cy.get('[data-test="form-errors"]')
+  }
```

Comment:
there already is a way to get this: `.errorSummary` (this same container), `.errorSummaryTitle` (“There is a problem”) and `.errorSummaryItems` (a list of the actual error `li` elements)